charts/aws-ingress/values.yaml

```diff
@@ -44,3 +45,21 @@ ingress:
     selector:
       key: app
       value: minio # (currently) fake-aws-s3 chart uses 'minio', minio-external chart uses 'minio-external'
+  teamSettings:
+    https:
+      externalPort: 443
+      sslCert: arn:aws:iam::00000-accountnumber-00000:server-certificate/example.com
+      sslPolicy: ELBSecurityPolicy-TLS-1-2-2017-01
+      hostname: <prefix>-teams-https.<domain>
+      ttl: 300
+    http:
+      teamSettingsPort: 8080
+  accountPages:
+    https:
+      externalPort: 443
+      sslCert: arn:aws:iam::00000-accountnumber-00000:server-certificate/example.com
+      sslPolicy: ELBSecurityPolicy-TLS-1-2-2017-01
+      hostname: <prefix>-account-https.<domain>
```

Comment:
Let's change these hostnames to be consistent with documentation: https://docs.wire.com/how-to/install/helm.html#how-to-set-up-dns-records.
I.e. use `example.com` instead of `<domain>` for easier grepping/chaging, and remove the `<prefix>`

(Also please apply the above to all items in aws-ingress/values.yaml)

Reply:
Good point, I changed a couple more references: https://github.com/wireapp/wire-server-deploy/commit/e1ebe2d429deed6172e09806859ce204db4d64e1